firestore-send-email/POSTINSTALL.md

```diff
@@ -62,6 +62,30 @@ Available properties for the `message` field are:
 * **text:** The plaintext content of the email.
 * **html:** The HTML content of the email.
 * **amp:** The [AMP4EMAIL][amp4email] content of the email.
+* **attachments:** An array containing attachment(s) as noted below.
+
+To send email attachments, you have to include an `attachments` array to the `message` map: 
+
+*Please note that the* `pathname` *property requires a URL path to the file you wish to attach.*
+*For further options, please consult [nodemailer attachment options](https://nodemailer.com/message/attachments/), bearing in mind you have no access to the file system.*
```

Comment:
Are all the options in this reference supported? https://nodemailer.com/message/attachments/

If so, I'd like to just point devs to that reference rather than add an example specifically for attachments.  Perhaps in a separate PR, we can add a more comprehensive example with all the options available.

I propose the entirety of the change for this PR be something like this instead:

**attachments:** An array containing attachment(s); supports [Nodemailer options](https://nodemailer.com/message/attachments/) (be aware that your email has no access to the recipient's file system)

Reply:
Not all those options are available for the user I'm afraid. I've updated the code to reflect your idea. I've also tweaked your suggestion from `recipient's file system` to `cloud server's file system`. I hope you don't mind!

Reply:
ah!  good to know!  Which ones _are_ supported?

Reply:
The following attachments are supported : "utf-8 string as an attachment", "define custom content type for the attachment", "use URL as an attachment", "encoded string as an attachment", "data uri as an attachment", "pregenerated MIME node". I've taken these from the code comments from here: https://nodemailer.com/message/attachments/ so you can easily reference each attachment type. If I had to hazard a guess, I would say the one user's really care about is the "use URL as an attachment" as this allows any file type attachment (.pdf, .txt, .doc, etc) provided it is hosted on the web and downloadable. All the others attachments are primitive.

Reply:
okie - added the supported types. PTAL?

As I mentioned in a previous comment in this thread, in a follow-up PR, let's add a full example for this message field (and we can use the URL option in that example).  But let's limit this PR to just adding the attachment support info to get the info out to devs.